modules/serverless-logic-openAPI-function-definition.adoc

```diff
@@ -28,9 +28,8 @@ The `operation` attribute is a `string` composed of the following parameters:
 
 {ServerlessLogicProductName} supports the following URI schemes:
 
-* classpath: Use this for files located in the `src/main/resources` folder of the application project. `classpath` is the default URI scheme. If you do not define a URI scheme, the file location is `src/main/resources/myopenapifile.yaml`.
```

Comment:
The reference to the class path must  be also removed from the sentence https://github.com/openshift/openshift-docs/pull/92779/files#diff-e6f50fdad600418c38217515274d8c0069fa2d534d5d2ab1dce31e98fa49b14fR25

`URI`: The engine uses this to locate the specification file, such as `classpath`.

should say:

`URI`: The engine uses this to locate the specification file.

Reply:
@wmedvede I removed the entire thing. Do you want me to add this?

Reply:
@kaldesai Walter references the line 25 in this file which should be fixed as he describes in his comment.

Reply:
@kaldesai basically, if you can find `classpath` anywhere in the OSL docs, we should remove/replace it accordingly

Reply:
@domhanak @wmedvede Got it! I have updated the content accordingly